packages/components/tsconfig.json

```diff
@@ -7,6 +7,6 @@
     "jsx": "react-jsx",
     "baseUrl": "."
   },
-  "include": ["src/index.tsx", "src/**/*.ts", "src/**/*.tsx"],
+  "include": ["src/index.tsx", "src/**/*.ts", "src/**/*.tsx", "../web/src/components/onboarding-modal.tsx"],
```

Comment:
Why this change? FWIW we cannot have files in the `components` package reference files outside it since it does not depend on web (and can't since that's circular dep). Also FWIW, we're going to remove the components package as that was introduced for reasons that no longer exist.